CHANGELOG.md

```diff
@@ -4,15 +4,67 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.10.0] - TBA
+## [0.10.0] - 2024-01-31
+
+### Highlights
+- The Zephyr CoAP library is now used instead of libcoap for Zephyr ports
+- Zephyr ports can specify a TLS credential tag to use for authentication,
+  in particular this will enable the use of offloaded DTLS sockets.
 
 ### Breaking Changes
 - Remove `golioth_` prefix from filenames and move header files under
   `golioth/` folder.
-- Stream: change `json` and `cbor` object set calls to specify type in
-  the parameters (instead of separate function calls). This matches the
-  approach taken in the LightDB State module:
-  `golioth_stream_set_async/sync`
+- Remove `golioth.h` header - users must now include individual headers
+  for each service that they use.
+- Stream and State: change `json` and `cbor` object set calls to specify
+  type in the parameters (instead of separate function calls).
+- RPC: This service must now be initialized prior to use by calling
+  `golioth_rpc_init(client);`
+- Settings: This service must now be initialzied prior to use by calling
+  `golioth_settings_init(client);`
+- Services (e.g. stream, fw_update, etc.) are disabled by default - be
+  sure to enable them in your project configuration.
+- Typedefs have been removed through the codebase - instead use the
+  underlying struct or enum types
```

Comment:
```suggestion
  underlying struct or enum types
- Remove `lightdb_` from `lightdb_stream` functions to differentiate from LIghtDB State
```

Reply:
Hmm, I thought I added that. Good catch!